src/lib/lumi.ts

```diff
@@ -2045,17 +2045,24 @@ export const lumiModernExtend = {
             ...args,
         }),
     lumiVibration: (): ModernExtend => {
-        const exposes: Expose[] = [e.action(['shake', 'triple_strike'])];
+        const exposes: Expose[] = [e.action(['triple_strike', 'movement'])];
 
         const fromZigbee: Fz.Converter[] = [
             {
-                cluster: 'ssIasZone',
```

Comment:
Maybe we should keep both converters? I'm not sure if this will break for others.